common/persistence/sql/domains.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS domains(
+  id VARCHAR(64) UNIQUE NOT NULL,
+  retention_days INT NOT NULL,
+  emit_metric TINYINT(1) NOT NULL,
+/* end domain_config */
+  config_version BIGINT NOT NULL,
+  db_version BIGINT,
```

Comment:
can use NOT NULL default 0 if that's what you want

Reply:
obsolete

Reply:
obsolete